clonelists/Nintendo - GameCube.json

```diff
@@ -450,6 +453,8 @@
 		],
 		"Nickelodeon SpongeBob SquarePants - The Movie": [
 			["Nickelodeon SpongeBob Schwammkopf - Der Film", 1]
```

Comment:
`Nickelodeon SpongeBob Schwammkopf - Der Film` should have a priority of `2`, so Retool knows which German title to pick first.

We want both `2 Games in 1 - Nickelodeon SpongeBob Schwammkopf - Der Film + Nickelodeon SpongeBob Schwammkopf - Schlacht um Bikini Bottom (Disc 1)` and `(Disc 2)` to become the chosen German titles in each of their respective situations, so people don't complain that there are missing discs.